packages/api/tests/rich-text-detection.test.ts

```diff
@@ -319,9 +325,12 @@ describe('detectFacets', () => {
           },
         ],
       ],
+      ['this #​ should not be a tag', [], []],
+      ['this #​a should not be a tag', [], []],
```

Comment:
Why isnt' this a tag? is there a zerowidth character there?

Reply:
Yep yep GitHub doesn't render them

Reply:
Perhaps using escape codes rather than the actual char itself would be beneficial here?